bin/ChakraCore/ChakraCore.vcxproj

```diff
@@ -148,7 +148,7 @@
     <ClInclude Include="TestHooks.h" />
   </ItemGroup>
   <!-- Conditionally include the group of files to sign because MBv2 knows what to do with this but the old build definition gets confused. -->
-  <ItemGroup Condition="'$(VSO_MICROBUILD_V2)'=='True'">
+  <ItemGroup Condition="'$(VSO_MICROBUILD_V2)'=='True' AND '$(POGO_TYPE)'!='PGI'">
```

Comment:
Any specific reason not to sign PGI? Is the signing process costly ?

I know we don't need it now, but we might if we plan to do pogo on arm.